lib/flags.mli

```diff
@@ -96,3 +96,6 @@ val default_inline_level : int
 (** Global profile_ltac flag  *)
 val profile_ltac : bool ref
 val profile_ltac_cutoff : float ref
+
+(** Default output directory *)
+val output_directory : CUnix.physical_path option ref
```

Comment:
IMO this should be in `system.ml` (use point), not here.

Reply:
Why not. I will consider it.